kola/tests/misc/files.go

```diff
@@ -173,6 +173,7 @@ func StickyDirs(c cluster.TestCluster) {
 		"/dev/mqueue",
 		"/dev/shm",
 		"/media",
+		"/run/torcx/unpack",
```

Comment:
If this doesn't need to be mode 1777, I'd rather the test fails until we can get the directory fixed.